tests/models/test_system.py

```diff
@@ -176,3 +175,56 @@ def test_add_source(self):
         self.system.add_source(source)
         self.assertEqual(len(self.system.sources.value), 1)
         self.assertEqual(source, self.system.sources.value[0])
+
```

Comment:
This was removed some time last week, check that you are up to date with main

Reply:
You are right, updating everything.